plugin/src/ignorePlaceIds.lua

```diff
@@ -0,0 +1,21 @@
+return {
```

Comment:
Add a comment explaining the purpose of this module (it could maybe use a more descriptive name too), then LGTM